openslides_backend/action/mixins/import_mixins.py

```diff
@@ -85,29 +209,27 @@ def on_success() -> None:
         return on_success
 
 
-class HeaderEntry(TypedDict):
-    property: str
-    type: str
-
-
 class StatisticEntry(TypedDict):
     name: str
     value: int
 
 
-class JsonUploadMixin(Action):
-    headers: List[HeaderEntry]
+class JsonUploadMixin(BaseImportJsonUpload):
+    headers: List[Dict[str, Any]]
```

Comment:
Why did you remove the typing here?

Reply:
It is only used here and Dict[str, Any] is easy to understand